config/config.go

```diff
@@ -63,12 +71,36 @@ func Get() Config {
 		printResponse = (val == "1" || val == "true")
 	}
 
+	printResponseBody := false
+	if val, exists := os.LookupEnv("print_response_body"); exists {
+		printResponseBody = (val == "1" || val == "true")
```

Comment:
Go has a method for this in the stdlib https://golang.org/pkg/strconv/#ParseBool

The faas-provider also has a method for this https://github.com/openfaas/faas-provider/blob/d6579bdcf7c85f4d01f398d65ea0cab37e9633d0/types/read_config.go#L60 which includes a default value. 

I would recommend using one of these. 

For reasons I can't remember the faas-provider does not use  https://golang.org/pkg/strconv/#ParseBool

Reply:
I'm keeping it consistent with the previous code.